scripts/migrate-flattened.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable camelcase */
+/* eslint-disable new-cap */
+/* eslint-disable no-multi-str */
+/* eslint-disable no-template-curly-in-string */
+
+require("dotenv").config()
+
+const exec = require("child_process").exec
+const fs = require("fs")
+const os = require("os")
+const cli = new cli_func()
+
+if (process.argv.length < 4) {
+  console.log()
+  console.log("\n\
+    Usage: yarn migrate-flattened <ArtifactName> <Network>\n\
+       or: npm run migrate-flattened <ArtifactName> <Network>\n\n\
+  ")
+  process.exit(0)
+}
+
+const artifact = process.argv[2]
+const network = process.argv[3]
+process.env.FLATTENED_DIRECTORY = `./flattened/${artifact}/`
+
+if (!fs.existsSync(`${process.env.FLATTENED_DIRECTORY}/Flattened${artifact}.sol`)) {
+  console.log("\n\
+    > Please, flatten the artifact first. E.g.:\n\
+      $ yarn flatten contracts" + (os.type() === "Windows_NT" ? "\\" : "/") + artifact + ".sol\n\n\
+  ")
+  process.exit(0)
```

Comment:
If a command hasn't done exactly what I asked it for, I'm expecting a non-zero exit code.
```suggestion
  process.exit(1)
```

Reply:
This script is thought to be invoked from a package manager (i.e. npm or yarn). If the script exists with a value different to 0, a lengthy and ugly message will be prompted to the user. By using `process.exit(0)` we assure the user gets a simple, clear and polite instruction message.